docs/BUILD.md

```diff
@@ -119,29 +119,29 @@ Clone the `openvino_tensorflow` repository:
   ```
 
 ###  2.1. <a name='BuildInstructions'></a>Build Instructions
-Use one of the following build options based on the requirements. **OpenVINO™ integration with TensorFlow** built using PyPI TensorFlow enable only the Python APIs, TensorFlow C++ libraries built from source is required to use the C++ APIs.
+Use one of the following build options based on the requirements. **OpenVINO™ integration with TensorFlow** built using PyPI TensorFlow enables only the Python APIs. TensorFlow C++ libraries built from source is required to use the C++ APIs.
 
-1. Pulls compatible prebuilt TF package from PyPi, clones and builds OpenVINO™ from source. The arguments are optional, if not provided then default versions as specified in build_ovtf.py will be used. 
+1. Pulls compatible prebuilt TensorFlow package from PyPi, clones and builds OpenVINO™ from source. The arguments are optional. If any argument is not provided, then the default versions as specified in build_ovtf.py will be used. 
 
         python3 build_ovtf.py --tf_version=v2.5.1 --openvino_version=2021.4.1
 
-2. Pulls compatible prebuilt TF package from PyPi. Uses OpenVINO™ binary from specified location.
+2. Pulls compatible prebuilt TensorFlow package from PyPi. Uses OpenVINO™ binary from specified location.
```

Comment:
Using abi 1, we pull the prebuilt TensorFlow from our GitHub repository. So, it's not in PyPi repository. Should we state that instead of saying "Pulls compatible prebuilt TensorFlow package from PyPi"?

Reply:
Right, need to change it. As we are pulling TF ABI1 from the github release assets